.github/workflows/dev-build.yml

```diff
@@ -190,13 +190,10 @@ jobs:
         with:
           repository: ${{ inputs.repository }}
           ref: ${{ inputs.commit }}
-          path: dev/greptimedb
-
-      - name: Enter greptimedb
-        run: |
-          cd dev/greptimedb
+          path: ${{ env.CHECKOUT_GREPTIMEDB_PATH }}
 
       - uses: ./.github/actions/build-linux-artifacts
+        working-directory: ${{ env.CHECKOUT_GREPTIMEDB_PATH }}
         with:
           arch: arm64
```

Comment:
question: arm64 artifacts need `path: ${{ env.CHECKOUT_GREPTIMEDB_PATH }}` ?

Reply:
Yes. The `working-directory` sets the target directory to build the greptimedb.